### PR TITLE
0.24 Support SSL certs in test helper rotation

### PR DESCRIPTION
**Pull request by FedericoCeratto**

### CODE REVIEW

File: af/analysis/rotation.py [hunk 1]
```diff
@@ -201,6 +241,136 @@ def drain_droplet_if_needed(db_conn, live_droplets, active_droplets_count):
...
+
+    cmd = [
+        "certbot",
+        "certonly",
+        "--dns-digitalocean",
+        "--dns-digitalocean-credentials",
+        certbot_creds,
+        "-d",
+        "'*.th.ooni.org'",
+    ]
```
Comment (hellais) [inline af/analysis/rotation.py:268]: Shouldn't this command be run in here somehow?

Comment (FedericoCeratto) [reply af/analysis/rotation.py:268]: It's commented out and there's a TODO for it

File: af/analysis/rotation.py [hunk 2]
```diff
@@ -64,9 +79,25 @@
 
 conffile_path = "/etc/ooni/rotation.conf"
 setup_script_path = "/etc/ooni/rotation_setup.sh"
+nginx_conf = "/etc/ooni/rotation_nginx_conf"
```
Comment (hellais) [inline af/analysis/rotation.py:82]: Why not embed the nginx configuration content into this script itself? If not maybe it should be documented somewhere how this looks like.

**Review by hellais — APPROVED** [review]
Left a minor nitpick comment about documenting how the nginx configuration looks like. Otherwise looks good to merge.

File: af/analysis/rotation.py (hunk 2 above)
Comment (FedericoCeratto) [reply af/analysis/rotation.py:82]: In order to store it in ansible.

**Pull request merged**